### PR TITLE
Migrate memcache client and modularization

### PR DESCRIPTION
**Pull request by plsmaop**
1. Since `github.com/youtube/vitess/go/memcache` is deprecated and deleted from the above repository
   (https://github.com/vitessio/vitess/issues/2446), the memcache client library used by `pttweb` should be replaced with
   `github.com/bradfitz/gomemcache` (written by the author of memcache)

2. Modern Go project should use go module

### CODE REVIEW

**Comment by wens** [comment]
Please split the replacement of the memcache client library and the addition of go module into separate commits.

**Comment by plsmaop** [comment]
> Please split the replacement of the memcache client library and the addition of go module into separate commits.

OK

File: cache/connpool.go [hunk 1]
```diff
@@ -106,7 +106,9 @@ func (m *MemcacheConnPool) manager() {
 }
 
 func (m *MemcacheConnPool) connect() {
-	if c, err := memcache.Connect(m.server, DefaultTimeout); err != nil {
+	c := memcache.New(m.server)
```
Comment (robertabcd) [inline cache/connpool.go:109]: This new library has an internal conn pool, but cannot limit the maximum open connections.
I think it's okay to layer ours on top of it.

Comment (robertabcd) [reply cache/connpool.go:108]: I'll move concurrency limiting into a separate abstraction layer later.
Let me absorb this PR first.

**Comment by robertabcd** [comment]
Thanks for contributing!

**Pull request merged**